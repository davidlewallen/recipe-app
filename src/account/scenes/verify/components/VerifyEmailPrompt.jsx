import React from 'react';
import Grid from 'react-bootstrap/lib/Grid';
import Panel from 'react-bootstrap/lib/Panel';

const VerifyEmailPrompt = () => (
  <Grid>
    <Panel>
      <Panel.Body className="align-center">
        To access your account, please follow the steps included in the email
        that was just sent to you.
      </Panel.Body>
    </Panel>
  </Grid>
);

export default VerifyEmailPrompt;
