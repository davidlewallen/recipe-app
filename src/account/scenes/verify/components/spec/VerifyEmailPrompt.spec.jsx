import React from 'react';
import { shallow } from 'enzyme';

import VerifyEmailPrompt from '../VerifyEmailPrompt';

describe('VerifyEmailPrompt', () => {
  it('should match snapshot', () => {
    expect(shallow(<VerifyEmailPrompt />)).toMatchSnapshot();
  });
});
