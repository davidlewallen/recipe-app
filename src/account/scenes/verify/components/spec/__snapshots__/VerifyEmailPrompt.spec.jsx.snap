// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerifyEmailPrompt should match snapshot 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <Uncontrolled(Panel)>
    <PanelBody
      bsClass="panel"
      className="align-center"
      collapsible={false}
    >
      To access your account, please follow the steps included in the email that was just sent to you.
    </PanelBody>
  </Uncontrolled(Panel)>
</Grid>
`;
